### ISSUE TITLE
Unable to extract yt initial data

### ISSUE THREAD

**Issue by x72x** [yt-dlp/yt-dlp#8062]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I'm running yt-dlp version **2023.07.06** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Germany

### Provide a description that is worded well enough to be understood

when am trying to download any video from youtube i got this errors, on all of my VPS ( Germany and Netherlands )

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
root@vmi1388531:~# yt-dlp -vU -f b https://youtu.be/8EzYoUxkg9c
[debug] Command-line config: ['-vU', '-f', 'b', 'https://youtu.be/8EzYoUxkg9c']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (pip)
[debug] Python 3.8.10 (CPython x86_64 64bit) - Linux-5.4.0-105-generic-x86_64-with-glibc2.29 (OpenSSL 1.1.1f  31 Mar 2020, glibc 2.31)[debug] exe versions: ffmpeg 4.2.7, ffprobe 4.2.7
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2019.11.28, mutagen-1.46.0, secretstorage-2.3.1, sqlite3-2.6.0, websockets-11.0.3
[debug] Proxy map: {}
[debug] Loaded 1855 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
yt-dlp is up to date (stable@2023.07.06)
[youtube] Extracting URL: https://youtu.be/8EzYoUxkg9c
[youtube] 8EzYoUxkg9c: Downloading webpage
WARNING: [youtube] unable to extract initial player response; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
[youtube] 8EzYoUxkg9c: Downloading ios player API JSON
[youtube] 8EzYoUxkg9c: Downloading android player API JSON
[youtube] 8EzYoUxkg9c: Downloading iframe API JS
[youtube] 8EzYoUxkg9c: Downloading player 7ee36b0e
[youtube] 8EzYoUxkg9c: Downloading web player API JSON
[youtube] 8EzYoUxkg9c: Downloading m3u8 information
WARNING: [youtube] unable to extract yt initial data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
WARNING: [youtube] Incomplete data received in embedded initial data; re-fetching using API.
[youtube] 8EzYoUxkg9c: Downloading initial data API JSON
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, size, br, asr, vext, aext, hasaud, id
[info] 8EzYoUxkg9c: Downloading 1 format(s): 22
[debug] Invoking http downloader on "https://rr5---sn-4g5lznl6.googlevideo.com/videoplayback?expire=1694199441&ei=MRr7ZIzzHfaZ6dsP1LO2sAE&ip=144.91.75.76&id=o-AH2OPcq9r7GvRpSF-NYQgwKChhTGAJ3A_oE83pXYEhdR&itag=22&source=youtube&requiressl=yes&mh=tP&mm=31%2C26&mn=sn-4g5lznl6%2Csn-5hne6nsy&ms=au%2Conr&mv=u&mvi=5&pl=24&gcr=de&spc=UWF9f68WW2w5apglf89nUBSHC7kcrIM&vprv=1&svpuc=1&mime=video%2Fmp4&cnr=14&ratebypass=yes&dur=399.151&lmt=1575362039116829&mt=1694177169&fvip=1&fexp=24007246%2C51000022&beids=24350018&c=ANDROID&txp=1311222&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cgcr%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Ccnr%2Cratebypass%2Cdur%2Clmt&sig=AOq0QJ8wRgIhAMJJnxTbte7T5PbzfmuIs9AdAtKBCkiMob75MMEmN758AiEAvwi9XlchSci3GIk_nBHkSW23SPQu9sY9RI4fgzPoOq0%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl&lsig=AG3C_xAwRQIgdb8gXtPkxeYb8nArGumcdrYBkBcGTwyErJEks9rBdssCIQCB2ik6iPrkPne1gh7y5vh6HHh9ewBJEeu0ENAAu8Zp4A%3D%3D"
ERROR: unable to download video data: HTTP Error 403: Forbidden
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/YoutubeDL.py", line 3363, in process_info
    success, real_download = self.dl(temp_filename, info_dict)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/YoutubeDL.py", line 3084, in dl
    return fd.download(name, new_info, subtitle)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/downloader/common.py", line 461, in download
    ret = self.real_download(filename, info_dict)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/downloader/http.py", line 377, in real_download
    establish_connection()
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/downloader/http.py", line 129, in establish_connection
    ctx.data = self.ydl.urlopen(request)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/YoutubeDL.py", line 3863, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
  File "/usr/lib/python3.8/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.8/urllib/request.py", line 640, in http_response
    response = self.parent.error(
  File "/usr/lib/python3.8/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.8/urllib/request.py", line 502, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.8/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: Forbidden

root@vmi1388531:~#
```

**Comment by bashonly** [comment]
Duplicate of #7860

tl;dr: your VPS' IP has been blocked by Youtube

workarounds:
 - try passing `-4` or `-6`
 - pass logged in youtube cookies (refreshed w/in the last 30 minutes)
 - change your IP or use a proxy
